### PR TITLE
Add tax return download link to client portal

### PR DESCRIPTION
**Pull request by bytheway875**
Screen Shot 2021-08-10 at 6 27 32 PM
Screen Shot 2021-08-10 at 6 27 51 PM

### CODE REVIEW

File: app/controllers/ctc/portal/portal_controller.rb [hunk 1]
```diff
@@ -9,10 +9,12 @@ def home
       @status = latest_transition.present? ? latest_transition.to_state : EfileSubmissionStateMachine.initial_state
       @exposed_error = latest_transition.present? ? latest_transition.client_facing_errors.first : nil
       @current_step = nil
+      @pdf1040 = current_client.documents.find_by(tax_return: submission.tax_return, document_type: DocumentTypes::Form1040.key)
```
Comment (alaroia-cfa) [inline app/controllers/ctc/portal/portal_controller.rb:12]: Should we be concerned about the possibility that there's more than one, for example, if the Hub user edits their data, then resubmits?

Not a blocker.

File: app/views/ctc/portal/portal/home.html.erb [hunk 2]
```diff
@@ -24,6 +24,12 @@
             <% if @current_step %>
               <%= link_to "Complete CTC form", @current_step, class: "button" %>
             <% end %>
+
+            <% if @pdf1040 %>
+              <%= link_to portal_document_path(id: @pdf1040.id), target: "_blank", rel: "no_opener", class: "button" do %>
```
Comment (alaroia-cfa) [inline app/views/ctc/portal/portal/home.html.erb:29]: `rel: "noopener"` not `rel: "no_opener"` but not a blocker

**Review by alaroia-cfa — APPROVED** [review]
Two questions and no blockers. Cheers!

File: app/controllers/ctc/portal/portal_controller.rb (hunk 1 above)
Comment (bytheway875) [reply app/controllers/ctc/portal/portal_controller.rb:12]: we use the same logic we use for creating new versions of the 136146, which is that we just say "if there is already a 1040 for this tax return, regenerate and reattach the file to the same document object". So there is only ever one!

**Pull request merged**